validator/src/lib.rs

```diff
@@ -395,9 +393,13 @@ fn validate_temporal<E: Interpreter + SuffixParams + Clone + Display>(
         }
         prev_action_and_timepoint = Some((action, timepoint));
     }
-    // If present, use the problem's epsilon which is smaller than the calculated one
+    // If present, use the problem's epsilon which is smaller than the calculated one.
+    // Otherwise, we just need to have a stricly positive epsilon to consider conditions
+    // between the existing actions. We take the calculated one divided by 10 to reduce it.
     if let Some(min_epsilon) = min_epsilon {
         env.epsilon = min_epsilon.clone();
```

Comment:
If the problem's epsilon is exactly the one that you computed (or close neough to it), couldn't this be problematic? (you wouldn't be able to fit your generated actions in between ?)

Reply:
The issue is to fit conditions between my generated actions.
If the problem's epsilon is exactly the one I've computed then we shouldn't be able to insert a condition between two actions separated by the epsilon, as this would mean that the epsilon is not respected.